src/server/kiwix-serve.cpp

```diff
@@ -161,9 +161,9 @@ void introduceTaskbar(string& content, const string& humanReadableBookId)
              humanReadableBookId,
              "__CONTENT__"));
     }
-    content = replaceRegex(content, rootLocation, "__ROOT_LOCATION__");
     content = replaceRegex(content, replaceRegex(humanReadableBookId, "%26", "&"), "__CONTENT_ESCAPED__");
```

Comment:
I would move the both lines out of the `if`.
Even if we don't need to replace `__CONTENT_ESCAPED__` for now, by moving the line out of the `if`, we prevent us against future bug when we will update static resources and add a `__CONTENT_ESCAPED__`